arbitrator/src/arbitrator.cpp

```diff
@@ -165,12 +165,18 @@ namespace arbitrator
         ros::shutdown(); // Will stop upper level spin and shutdown node
     }
 
-    void Arbitrator::pose_cb(const geometry_msgs::PoseStampedConstPtr& msg) 
+    void Arbitrator::bumper_pose_cb()
     {
-        vehicle_state_.stamp = msg->header.stamp;
-        lookupFrontBumperTransform();   
-        
-        
+        try
+        {
+            tf_ = tf2_buffer_.lookupTransform("map", "vehicle_front", ros::Time(0), ros::Duration(1.0)); //save to local copy of transform 20 sec timeout
```

Comment:
Where is this 20s timeout defined? It looks like 1.0s timeout on the lookup itself, is there somewhere else a 20 second timeout (on data validity?) is set?

Reply:
It is arbitrary, not required.